content/en/blog/2024/otel-docs-survey/index.md

```diff
@@ -0,0 +1,152 @@
+- Better structure: 8 (17%)
+- Add code repository docs: 5 (10%)
+- Other: 2 (4%)
+- No response: 7 (15%)
+
+![Horizontal bar graph showing the number of feature requests by category](feature-request.png)
+
+#### Q: Would more visual aids (e.g., diagrams and screenshots) explaining OpenTelemetry concepts be helpful?
+
+An overwhelming 81% said yes: they want more visual aids.
+
+### Current state of the docs
+
+#### Q: How well do the current docs at opentelemetry.io explain the different components of OpenTelemetry?
+
+Most respondents felt the component conceptual documentation was average, with a
+top score of 3.
+
+![Stacked row chart showing the percentage of users who rated the component conceptual documentation from 1 (low) to 5 (high)](component-explanations.png)
+
+#### Q: How straightforward and user-friendly are the installation instructions for OpenTelemetry?
+
+Most respondents found OTel instructions better than average, with a top score
+of 4. Respondents with intermediate-level observability knowledge rated them
+higher than experts: 55% of intermediates rated the installation instructions 4
+or 5, compared with only 17% of experts.
+
+![Stacked row chart showing the percentage of users who rated the installation instructions from 1 (low) to 5 (high)](installation-instructions.png)
+
+#### Q: How comprehensive are the troubleshooting sections?
+
+Most respondents believe that this section of docs needs work. Only 15% rated
+the troubleshooting docs 4 or 5, and they were all intermediate-level
+respondents. None of the expert-level respondents rated the troubleshooting docs
+above a 3.
+
+![Stacked row chart showing the percentage of users who rated the troubleshooting documentation from 1 (low) to 5 (high)](troubleshooting.png)
+
+#### Q: How would you rate your experience using the current OTel documentation for the following languages and components?
+
+Respondents were asked to rate only the documentation that applied to them, so
+we can infer based on their responses which docs sets are the most used.
+
+- The Collector documentation is the most used: 77% of respondents rated it.
+- The next five documentation sets are close in popularity, with 50 to 67% of
+  respondents rating them.
+
+![Horizontal bar graph showing the number of ratings per OTel component](top-six-bar.png)
+
+Here are the tabulated ratings for all languages and components. When the
+results are normalized and weighted, we can see additional insights:
+
+- The Java documentation has the highest overall rating.
+- The Swift documentation has the lowest overall rating.
+
+![Table showing the number of ratings (poor, okay, great) by OpenTelemetry documentation set](ratings-table.png)
+
+If we combine these insights, we can see that the documentation used by the most
+people that needs the most work is the JavaScript documentation.
+
+![Subset of the previous table showing the ratings for the six most popular documentation sets](javascript-table.png)
```

Comment:
Again, how about using a markdown table here?

```suggestion
> **Where should we focus our improvement efforts?**<br> _JavaScript is one of
> the six most-used docs sets, but its rating is the lowest._
>
> | Language or component | Poor | Okay | Great | Total responses | Normalized & weighted |
> | :-------------------- | :--: | :--: | :---: | :-------------: | :-------------------: |
> | GO                    |  6   |  12  |   9   |       27        |        7.1111         |
> | Java                  |  3   |  16  |   8   |       27        |        7.3333         |
> | JavaScript            |  3   |  19  |   2   |       24        |        6.0000         |
> | Python                |  6   |  17  |   8   |       31        |        6.7742         |
> | Collector             |  9   |  17  |  11   |       37        |        6.8108         |
> | Kubernetes            |  6   |  20  |   6   |       32        |        6.3750         |
```

If you agree, drop the corresponding image. Here's what it looks like:

<img width="664" alt="image" src="https://github.com/user-attachments/assets/b9ade371-e1a9-4b9a-9cb1-af87d405d284" />